tokio-test/src/macros.rs

```diff
@@ -0,0 +1,41 @@
+//! A collection of useful macros for testing futures and tokio based code
+
+/// Assert if a future is ready
```

Comment:
Nit: this is really asserting that a _poll_ is ready --- if we were asserting that a future was ready, we would call `poll` on `$e`...